tests/mlmodel_openai/test_embeddings_error_v1.py

```diff
@@ -12,17 +12,305 @@
+    }
+)
+@validate_transaction_metrics(
+    name="test_embeddings_error_v1:test_embeddings_invalid_request_error_no_model",
+    scoped_metrics=[("Llm/embedding/OpenAI/create", 1)],
+    rollup_metrics=[("Llm/embedding/OpenAI/create", 1)],
+    custom_metrics=[
+        ("Python/ML/OpenAI/%s" % openai.__version__, 1),
+    ],
+    background_task=True,
+)
+@validate_custom_events(no_model_events)
+@validate_custom_event_count(count=1)
+@background_task()
+def test_embeddings_invalid_request_error_no_model(set_trace_info, sync_openai_client):
+    with pytest.raises(TypeError):
+        set_trace_info()
+        sync_openai_client.embeddings.create(input="This is an embedding test with no model.")  # no model provided
+
+
+@dt_enabled
+@reset_core_stats_engine()
+@validate_error_trace_attributes(
+    callable_name(TypeError),
+    exact_attrs={
+        "agent": {},
+        "intrinsic": {},
+        "user": {},
+    },
+)
+@validate_span_events(
+    exact_agents={
+        "error.message": "create() missing 1 required keyword-only argument: 'model'"
+        if sys.version_info < (3, 10)
+        else "AsyncEmbeddings.create() missing 1 required keyword-only argument: 'model'",
+    }
+)
+@validate_transaction_metrics(
+    name="test_embeddings_error_v1:test_embeddings_invalid_request_error_no_model_async",
+    scoped_metrics=[("Llm/embedding/OpenAI/create", 1)],
+    rollup_metrics=[("Llm/embedding/OpenAI/create", 1)],
+    custom_metrics=[
+        ("Python/ML/OpenAI/%s" % openai.__version__, 1),
+    ],
+    background_task=True,
+)
+@validate_custom_events(no_model_events)
+@validate_custom_event_count(count=1)
+@background_task()
+def test_embeddings_invalid_request_error_no_model_async(set_trace_info, async_openai_client, loop):
+    with pytest.raises(TypeError):
+        set_trace_info()
+        loop.run_until_complete(
+            async_openai_client.embeddings.create(input="This is an embedding test with no model.")
+        )  # no model provided
+
+
+invalid_model_events = [
+    (
+        {"type": "LlmEmbedding"},
+        {
+            "id": None,  # UUID that varies with each run
+            "appName": "Python Agent Test (mlmodel_openai)",
+            "transaction_id": "transaction-id",
+            "span_id": None,
+            "trace_id": "trace-id",
+            "input": "Model does not exist.",
+            "api_key_last_four_digits": "sk-CRET",
+            "duration": None,  # Response time varies each test run
+            "request.model": "does-not-exist",
+            "response.organization": None,
+            "vendor": "openAI",
+            "ingest_source": "Python",
+            "error": True,
+        },
+    ),
+]
+
+
+@dt_enabled
+@reset_core_stats_engine()
+@validate_error_trace_attributes(
+    callable_name(openai.InternalServerError),
+    exact_attrs={
+        "agent": {},
+        "intrinsic": {},
+        "user": {},
```

Comment:
I'd expect a minimum to have a status code in here but also usually the error.code is populated as well. I'm guessing these were removed cause the api was miss-behaving yesterday but since we are mocking these should be consistent.
```suggestion
        "user": {
            'http.statusCode': 404,
            'error.code': 'model_not_found',
        },
```